runtime/autoload/dist/ft.vim

```diff
@@ -639,13 +639,14 @@ export def FTmms()
   setf mmix
 enddef
 
-# This function checks if one of the first five lines start with a dot.  In
-# that case it is probably an nroff file: 'filetype' is set and 1 is returned.
+# This function checks if one of the first ten lines start with a typical
+# nroff pattern in man files.  In that case it is probably an nroff file:
+# 'filetype' is set and 1 is returned.
 export def FTnroff(): number
   var n = 1
-  while n < 5
+  while n <= 5
```

Comment:
did you mean to check the first 10 lines or the first 5 lines? Please adjust either the comment or the actual limit here then.

Reply:
My bad, the comment was wrong.